wp-content/themes/oceanwp-child/functions.php

```diff
@@ -259,4 +260,70 @@ function add_related_news_to_initiative_pages() {
          
     }
  
+}
+
+
+add_action( 'wp_ajax_follow_post', 'follow_initiative' );
+function follow_initiative() {
+	global $wpdb; 
+    
+        
+	if ( !is_user_logged_in() ) {
+
+		wp_redirect('http://localhost/ggrc_website/'); 
+		
+		exit;
+	}
+	else{
+	
+		$current_user_id = get_current_user_id();
+		$postid = $_POST['postid'];
+
+		//echo $current_user_id;
+
+			
+		$table_name = $wpdb->prefix . 'follow_posts';     
+		$wpdb->insert($table_name, array('userID' => $current_user_id, 'postID' => $postid)); 
+		
+		
+	}
+
+}
+
+add_action( 'wp_ajax_unfollow_post', 'unfollow_initiative' );
+function unfollow_initiative() {
+	global $wpdb;     
+        
+	if ( !is_user_logged_in() ) {
+
+		wp_redirect('http://localhost/ggrc_website/'); 
+		
+		exit;
+	}
+	else{
+	
+		$current_user_id = get_current_user_id();
+		$postid = $_POST['postid'];
+
+		//echo $current_user_id;
+			
+		$table_name = $wpdb->prefix . 'follow_posts';     
+		 
+		$wpdb->query($wpdb->prepare("UPDATE $table_name SET isFollowing = '0' WHERE userID = '$current_user_id' and postID = '$postid'"));		
+		
+	}
+
+}
+
+//add_action( 'check_follow', 'check_following' );
+
+function check_following() {
```

Comment:
I'm having a hard time understanding what this function is doing.
Could you please change the name to be meaningful. Something like: `countInitiativeFollowers` maybe?

Reply:
Function is checking if a user is following a certain initiative post. Changed the name to checkIsFollowing();